code/workspace/linting-and-testing/example_sqs_queue.yaml

```diff
@@ -0,0 +1,25 @@
+AWSTemplateFormatVersion: "2010-09-09"
+
+Description: AWS CloudFormation workshop - linting and testing - example SQS queue (uksb-1q9p31idr)
+
+Resources:
+  MyQueue:
+    Type: AWS::SQS::Queue
+    Properties:
+      # Use `cfn-lint` to find issues on this example template, and
+      # then fix issues you found
+      DelaySeconds: 1000
+      Tag:
+        - Key: example_key_1
+          Value: example_value_1
+        - Key: example_key_2
+          Value: example_value_2
```

Comment:
Can we use better key/value names? For example:
```
- Key: Name
  Value: workshop-sqs-queue
```

Reply:
Sure!  Will update.